src/Controller.php

```diff
@@ -135,6 +143,35 @@ public function postPublish($group = null)
 
         $this->manager->exportTranslations($group, $json);
 
-        return ['status' => 'ok'];
-    }
+		return ['status' => 'ok'];
+	}
+
+	/**
+	 * @return array
+	 */
+	public function postSaveKeyJson()
```

Comment:
💡 return typehinting

Reply:
must be support php 5.6, be careful with the type hinting

Reply:
must be support php 5.4, be careful with the type hinting.